src/main/java/tel/schich/javacan/BcmFlag.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * The MIT License
+ * Copyright © 2018 Phillip Schichtel
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package tel.schich.javacan;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+/**
+ * The BcmFlag enum represent the Broadcast Manager message flags.
+ *
+ * @see https://www.kernel.org/doc/html/latest/networking/can.html#broadcast-manager-message-flags
+ */
+public enum BcmFlag {
+
+    /**
+     * Set the values of ival1, ival2 and count
+     */
+    SETTIMER(0x0001),
```

Comment:
I like the binary (`0b00000001` with separators between bytes) notation for bitmask stuff, quicker to read, compare and modify. See `CanId`

Reply:
If you insist i will change it, but i think for single bit flags the hex notation is more readable.

Especially as this class is basically a copy of the flags from the Linux header file, where the hex notation is used. If i translate this into the binary notation it is harder to compare to the original header file than now. So i would prefer it to remain as hex, like in the Linux header.

Reply:
The symmetry with the kernel code is a good point.